src/types.ts

```diff
@@ -147,4 +147,52 @@ export interface ManifestOptions {
    * @default A combination of `routerBase` and `options.build.publicPath`
    */
   publicPath: string
+  /**
+   * @default []
+   */
+  related_applications: {
```

Comment:
should be `related_applications?: {...}[]` [spec](https://developer.mozilla.org/en-US/docs/Web/Manifest/prefer_related_applications)